.github/workflows/set-milestone-from-base-branch.yml

```diff
@@ -15,14 +15,14 @@ jobs:
           PR_NUMBER: ${{ github.event.number }}
         name: "Set milestone on PR based on base branch"
         run: |
-          readarray -t milestones < <(gh api /repos/brave/brave-core/milestones -q '.[] | .title')
+          readarray -t milestones < <(gh api /repos/brave/brave-core/milestones -q ".[] | .title")
           if [ "$GITHUB_BASE_REF" = "master" ]; then
               milestone_selector="Nightly"
           else
               milestone_selector="$GITHUB_BASE_REF"
           fi
 
-          milestone=$(printf "%s\n" "${milestones[@]}" | grep -E "$milestone_selector")
+          milestone=$(printf "%s\n" "${milestones[@]}" | grep -F "$milestone_selector")
```

Comment:
Could we also add `-x` to `grep`, to match the complete line?

Reply:
I don't think this works, bec the selector is like `1.38.x` while the milestone is longer. So by definition it's a partial match.

Reply:
The partial matches make me uneasy, but the simplicity is nice vs. full matches using regular expressions. Up to you.